### PR TITLE
Sharing feature

### PR DESCRIPTION
**Pull request by PaulJKim**
**Asana task**: [[Screenplay] Implement sharing feature](https://app.asana.com/0/1185117109217432/1204083894737784/f)

Description
This PR adds some new buttons and a dropdown that encompass a new action bar for screen details within a place row. Along with these, there are also some added tooltips and a toast that appears when the link is copied to the user's clipboard. Designs are linked in the Asana Task.

### CODE REVIEW

**Review by hannahpurcell — CHANGES_REQUESTED** [review]
Couple of visual deviations, before I get into the nitty gritty code review: 

The tooltips should be centered on the icon, but are left-aligned:
Screenshot 2023-06-12 at 8 03 49 AM

The logic by which the buttons collapse into the "More" icon seems buggy. Attaching an instance where it should have collapsed but wrapped instead, and another instance where it collapsed, but also seems that it should have had room to not collapse? Or maybe we always want to collapse them for the countdown clocks?

Screenshot 2023-06-12 at 8 01 04 AM

Screenshot 2023-06-12 at 8 03 20 AM

**Comment by hannahpurcell** [comment]
Ohhhhh I see I see. I interpreted the "more" button to be a collapse situation, but it doesn't need to be. It can be specific to screen type / number of screens. Works for me

**Comment by hannahpurcell** [comment]
Ohhhhh I see I see. I interpreted the "more" button to be a collapse situation, but it doesn't need to be. It can be specific to screen type / number of screens. Works for me

**Review by hannahpurcell — COMMENTED** [review]
Ok great! Two cosmetic edits, and then the one visual tweak I already mentioned.

Plus, the toast needs some styling updates. To name a few (not comprehensive list):
- SVG checkmark should be 24px, not 18px
- Margin between svg and text should be bigger, I think
- Shouldn't have a dark border
- Should have a darker, thicker shadow

Screenshot 2023-06-12 at 10 31 39 AM

[Figma](https://www.figma.com/file/w40fdAbU6EgnydfOLtBXxI/MorePolished_AccessWebScreens?type=design&node-id=4961-45405&t=cipgbddRthpXjT55-4):
Screenshot 2023-06-12 at 10 30 42 AM

**Comment by PaulJKim** [comment]
Hmm interesting, I'm not able to replicate the misaligned tooltip 🤔 
Screenshot 2023-06-13 at 2 24 52 PM

File: assets/js/components/Dashboard/ScreenDetailActionBar.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  FlagFill,
+  BoxArrowUpRight,
+  ThreeDotsVertical,
+} from "react-bootstrap-icons";
+import { useScreenplayDispatchContext } from "../../hooks/useScreenplayContext";
+
+interface ScreenDetailActionBarProps {
+  screenUrl: string;
+  isPaess?: boolean;
+  isMultipleScreens?: boolean;
```
[the remark this answers is not in the document]
Comment (PaulJKim) [reply assets/js/components/Dashboard/ScreenDetailActionBar.tsx:17]: Oh yes agreed. I think that's much simpler!

**Comment by hannahpurcell** [comment]
Hmmmph.... I did reproduce the uncentered tooltip again, in both Chrome and Firefox. If you try to reproduce it once more and still cannot, I'd say either phone a friend, or we can merge it and look at it in dev

**Pull request merged**